src/site/markdown/mod.rs

```diff
@@ -9,10 +9,14 @@ use std::collections::HashMap;
 use std::fs;
 use std::path::Path;
 
-pub struct Adapters {}
-impl SyntaxHighlighterAdapter for Adapters {
+use self::syntax_highlight::syntax_themes::SyntaxThemes;
+
+pub struct Adapters<'a> {
+    syntax_theme: &'a SyntaxThemes,
```

Comment:
we might want to rename the enum to the singular `SyntaxTheme`

Reply:
done!